config/production.yml

```diff
@@ -2,8 +2,9 @@ accounts:
   passwordResetEmailEnabled: true
 
 defaultOTP: '1111'
+defaultPhoneNumber: '9999999999'
 
-enableSMS: false
+enableSMS: true
```

Comment:
Can we refactor this config to:

```yaml
sms:
  enabled: true
```

instead?

Reply:
this looks good, modified it!